### DISCUSSION TITLE
Serve in subfolder?

### DISCUSSION THREAD

**Discussion by sbscan** [pocketbase/pocketbase#4376] [Q&A]
I'm using OLS (like NGINX) for reverse proxy and want to serve under a subfolder like 'mydomain.com/pocketbase/_'. I couln't find a base path setting, so pocketbase gives not found error as below. Any setting, or should I compile myself?

ERROR GET /pocketbase
└─ Not Found.
   └─ code=404, message=Not Found

**Comment by ganigeorgiev** [comment]
PocketBase doesn't have "base path" or similar option.

I'm not familiar with OLS but if it is similar to NGINX you should be able to register a `rewrite` rule as shown in the [NGINX minimal example](https://pocketbase.io/docs/going-to-production/#using-reverse-proxy).
